src/helpers/shareHelper.js

```diff
@@ -38,20 +39,26 @@ export const openShare = async ({ message, title, url }) => {
 };
 
 export const shareMessage = (data, query) => {
+  const urls = mergeWebUrls({
+    webUrls: data.webUrls,
+    contact: data.contact,
+    contacts: data.contacts
+  })?.map(({ url }) => `\n${url}`);
+
   const buildMessage = (query) => {
     switch (query) {
       case QUERY_TYPES.EVENT_RECORD:
         return `${momentFormat(data.listDate)} | ${
           data.addresses?.[0]?.addition || data.addresses?.[0]?.city
-        }: ${data.title}`;
+        }: ${data.title}${`\n${urls}`}`;
       case QUERY_TYPES.NEWS_ITEM:
         return `${momentFormat(data.publishedAt)} | ${data.dataProvider?.name}: ${
           data.contentBlocks?.[0]?.title
-        }`;
+        }${`\n${urls}`}`;
       case QUERY_TYPES.POINT_OF_INTEREST:
-        return `${data.category?.name}: ${data.name}`;
+        return `${data.category?.name}: ${data.name}${`\n${urls}`}`;
       case QUERY_TYPES.TOUR:
-        return `${data.category?.name}: ${data.name}`;
+        return `${data.category?.name}: ${data.name}${`\n${urls}`}`;
```

Comment:
please do not add line breaks if there are no urls. this causes an additional empty line in the share message.

Reply:
Yes, that's right. I overlooked that. I've edited it now.
https://github.com/smart-village-solutions/smart-village-app-app/pull/916/commits/081c27876bf3cbd07a86a776fd5aff7f5aef0b28

Reply:
did you ensure, that rendering will not result in "...false" if there are no urls?

Reply:
we have 3 different situations. One is the case of more than one link, the second is the case of only one link and the third is the case of no link at all. All three cases look like in the screenshot.
<img width="245" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/1a307158-e4a8-4e41-904f-f293f14f97da">

Reply:
In my eyes the last one hast two empty lines?

Reply:
Yes, you're right. That's not supposed to happen. I'm trying to solve this problem. 👍

Reply:
I fixed the problem and now the 3 different options that can occur when the share button is pressed are as in the screenshot.
<img width="250" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/3f8e82e7-8e4f-48ef-a2d4-6e3edcd34f7e">

https://github.com/smart-village-solutions/smart-village-app-app/pull/916/commits/a65c1e0cfa0acb3da5fa37faa921a6e5f2cad86a

Reply:
There is a } at the end of every title now

Reply:
Yes, that's right. I fixed that problem.

<img width="246" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/b473751e-8ec6-46cd-a75f-32425c629cdb">

https://github.com/smart-village-solutions/smart-village-app-app/commit/ab5b6bec9d66d8aab4869a5b39603472b288d1b1